packages/evidence/cli.js

```diff
@@ -314,4 +314,27 @@ prog
 		});
 	});
 
+prog
+	.command('preview')
+	.describe('preview the production build')
+	.action((args) => {
+		if (args.debug) {
+			process.env.VITE_EVIDENCE_DEBUG = true;
+			delete args.debug;
+		}
+
+		const flatArgs = flattenArguments(args);
+
+		// Run svelte kit dev in the hidden directory
+		const child = spawn('npx vite preview --outDir build --port 3000', flatArgs, {
```

Comment:
@csjh that's why I'm explicitly passing the --outdir flag

When I delete the `build` directory, this process throws

Reply:
Huh, didn't realize vite supported them like that, nice